docs/simulation/calculations.yaml

```
@@ -51,7 +51,44 @@ Economy:
       formula: MGQ * GOLD_TO_CASH_RATE
       MGQ: Money good quanity = total production of goods with money = yes
       GOLD_TO_CASH_RATE: defines.country.GOLD_TO_CASH_RATE
+    Overseas maintenance:
+      See: https://github.com/schombert/Project-Alice/blob/main/docs/rules.md#overseas-penalty
   Production:
+    Resource gathering operation:
+      base workforce: defined in common/production_types.txt as workforce
+      rgo size:
+        formula: floor(1.5*ceil(n_workers_in_state / (base workforce * (1 + terrain rgo size modifier)))
```

```suggestion
        formula: floor(1.5 * ceil(n_workers_in_state / (base workforce * (1 + terrain rgo size modifier)))
```
Consistent spacing would look cleaner